packages/core/src/external/fhir/patient/index.ts

```diff
@@ -63,14 +63,19 @@ export const toFHIR = (patient: Pick<Patient, "id" | "data">): FHIRPatient => {
   };
 };
 
-const convertDriversLicenseToIdentifier = (
-  personalIdentifiers: PersonalIdentifier[]
-): Identifier[] => {
+export const convertPIToIdentifier = (personalIdentifiers: PersonalIdentifier[]): Identifier[] => {
```

Comment:
Since we're touching this, can we move it to a regular function, please? [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#b669ec3473f1414b9d234cbdac20041b)

Also, the name is a bit confusing (requires one to open the function to see what's doing): `PI` reads as the Pi number. 😄 

And we've been moving to CamelCase even for acronyms - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#2181aa6e30ae46ccb8a33f8ce1e0dbc0), which would make that relation with the Pi number even worse.

Maybe something like `personalIdentifierToIdentifier()`?

Reply:
Done.